shoes-package/spec/spec_helper.rb

```diff
@@ -12,7 +12,7 @@
 require 'pathname'
 require 'rspec/its'
 
-Dir["#{SHOES_PACKAGE_SPEC_ROOT}/support/**/*.rb"].each {|f| require f}
+Dir["#{SHOES_PACKAGE_SPEC_ROOT}/support/**/*.rb"].each { |f| require f }
```

Comment:
I also prefer no space before my block parameters. Looks like there's a config option for this too, `SpaceBeforeBlockParameters`. I'll drop this commit and do a new PR for this too.